### DISCUSSION TITLE
OTG showing folders but no files inside

### DISCUSSION THREAD

**Discussion by justincampbell73** [FossifyOrg/General-Discussion#240] [General]
Just downloaded app because all other foss file mangers made me go back to android file manager a mount USB each time I wanted to use which is annoying. I open fossify file manager and was happy to see it didn't make me do that, but it shows my folders in otg and does not show files inside the folders. it showed one image file but that's it, no roms or apks.

**Comment by Aga-C** [comment]
Please create a bug report in [File Manager](https://github.com/FossifyOrg/File-Manager/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=bug_report.yml).
